pkg/server/chaosd/disk.go

```diff
@@ -196,29 +182,56 @@ func (diskAttack) diskFill(fill *core.DiskOption) error {
 			log.Error("fail to get disk total size", zap.Error(err))
 			return err
 		}
-		fill.Size = strconv.FormatUint(totalSize*percent/100, 10)
+		fill.Size = strconv.FormatUint(totalSize*percent/100, 10) + "c"
```

Comment:
Why need to add `c` here?

Reply:
`c` is a byte size unit in [dd](https://man7.org/linux/man-pages/man1/dd.1.html) means one byte.
`       N and BYTES may be followed by the following multiplicative
       suffixes: c=1, w=2, b=512, kB=1000, K=1024, MB=1000*1000,
       M=1024*1024, xM=M, GB=1000*1000*1000, G=1024*1024*1024, and so on
       for T, P, E, Z, Y.  Binary prefixes can be used, too: KiB=K,
       MiB=M, and so on.`